codechecker_lib/session_manager.py

```diff
@@ -305,29 +310,22 @@ def __init__(self):
         LOG.debug('Loading session config')
 
         # Check whether user's configuration exists.
-        session_cfg_file = os.path.join(os.path.expanduser("~"),
+        user_home = os.path.expanduser("~")
+        session_cfg_file = os.path.join(user_home,
                                         ".codechecker_passwords.json")
-        if not os.path.exists(session_cfg_file):
-            LOG.info("CodeChecker authentication client's example "
-                     "configuration file created at " + session_cfg_file)
-            shutil.copyfile(os.path.join(os.environ['CC_PACKAGE_ROOT'],
-                                         "config", "session_client.json"),
-                            session_cfg_file)
-            os.chmod(session_cfg_file, stat.S_IRUSR | stat.S_IWUSR)
-
         LOG.debug(session_cfg_file)
 
         scfg_dict = load_session_cfg(session_cfg_file)
 
-        if not scfg_dict["credentials"]:
+        if not scfg_dict.get("credentials"):
             scfg_dict["credentials"] = {}
 
         self.__save = scfg_dict
         self.__autologin = scfg_dict.get("client_autologin") \
             if "client_autologin" in scfg_dict else True
 
         # Check and load token storage for user
-        self.token_file = os.path.join(tempfile.gettempdir(), ".codechecker_" +
+        self.token_file = os.path.join(user_home, ".codechecker_" +
                                        getpass.getuser() + ".session.json")
         LOG.debug(self.token_file)
         if os.path.exists(self.token_file):
```

Comment:
I moved out to a separate function the permission checking so its better now than before. We can check the permissions for both files. Mainly we should warn about the config file because it needs to be created manually by the user. The token file is handled automatically with the right permissions, but I've put a warning there too, just in case.